css-pseudo-4/Overview.bs

```diff
@@ -616,9 +657,10 @@ Default UA Styles</h3>
 <h3 id=highlight-bounds>
 Area of a Highlight</h3>
 
-  <p>For each type of highlighting (see [[#highlight-selectors]])
-  there exists a single <dfn>highlight overlay</dfn> for the entire document,
-  the active portions of which are represented
+  <p>For each type of highlighting except for ''::highlight'' (see [[#highlight-selectors]])
+  there exists a single <dfn>highlight overlay</dfn> for the entire document. For ''::highlight''
+  there exists one overlay for each [=custom highlight=] in the document.
+  The active portions of each overlay are represented
```

Comment:
If we consider each named highlight a different type of highlight (which we should, imho) then this original sentence remains correct.

```suggestion
  <p>For each type of highlighting (see [[#highlight-selectors]])
  there exists a single <dfn>highlight overlay</dfn> for the entire document,
  the active portions of which are represented
```